libs/net/dal/Migrations/1.3.26/Up/PostUp/00-ReportTemplate.sql

```diff
@@ -0,0 +1,374 @@
+                <a href="@($"{ViewContentUrl}{content.Id}")" target="_blank" style="color: #6750a4; text-decoration: none;">View Article <img height="14" style="max-width: 14px; height: 14px;" valign="absmiddle" src="@($"{SubscriberAppUrl}assets/reports/follow_link.png")"></a>
+              </span>
+            </div>
+          }
+          else if (Settings.Content.ShowLinkToStory)
+          {
+            <div>
+              <span style="font-size: .85em; text-transform:uppercase; vertical-align: middle; background-color:#FFF; border:#ccc 1px solid; margin:20 auto; padding:6px 10px;">
+                <a rel="noreferrer" href="@($"{sourceUrl}")" target="_blank" style="color: #6750a4; text-decoration: none;">View Article (external site)<img height="14" style="max-width: 14px; height: 14px;" valign="absmiddle" src="@($"{SubscriberAppUrl}assets/reports/follow_link.png")"></a>
+              </span>
+            </div>
+          }
+          <div style="text-align: right; margin-top: 20px;">
+            <a href="#item-@itemPosition" style="background-color:#6750A4; color:#FFFFFF; padding:5px 10px; text-decoration:none; border-radius:5px;">To Article Top</a>
+          </div>
+
+        }
+      }
+    }
+    else if (section.Value.SectionType == ReportSectionType.MediaAnalytics)
+    {
+      @* Media Analytics Section *@
+      foreach (var chart in section.Value.ChartTemplates)
+      {
+        <img alt="@chart.SectionSettings.AltText" src="@chart.Uid" />
+      }
+    }
+    else if (section.Value.SectionType == ReportSectionType.Gallery)
+    {
+      @* Gallery Section *@
+      if (section.Value.Settings.Direction == "row" && section.Value.Settings.ShowImage)
+      {
+        <div style="display:flex;flex-wrap:wrap;">
+          @for (var i = 0; i < sectionContent.Length; i++)
+          {
+            var content = sectionContent[i];
+            var fileName = content.FileReferences.FirstOrDefault()?.FileName ?? content.Id.ToString();
+            var src = $"data:{content.ContentType};base64," + content.ImageContent;
+
+            <img style="height:min-content" src="@src" alt="@fileName" />
+          }
+        </div>
+      }
+      else if (section.Value.Settings.ShowImage)
+      {
+        for (var i = 0; i < sectionContent.Length; i++)
+        {
+          var content = sectionContent[i];
+          var fileName = content.FileReferences.FirstOrDefault()?.FileName ?? content.Id.ToString();
+          var src = $"data:{content.ContentType};base64," + content.ImageContent;
+          if (!string.IsNullOrEmpty(content.ImageContent))
+          {
+            <div>
+              <img style="height:min-content" src="@src" alt="@fileName" />
+            </div>
+          }
+        }
+      }
+    }
+    else if (section.Value.SectionType == ReportSectionType.Image)
```

Comment:
View the accompanying CustomReport.cshtml file to see this addition.  We can now display Image sections.